scripts/scil_compare_connectivity.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Performs a network-based statistical comparison for populations X and Y using
+a t-statistic threshold of alpha. All matrices must have the same shape and be
+ordered in the same way.
+
+For more details visit:
+https://brainconn.readthedocs.io/en/latest/generated/brainconn.nbs.nbs_bct.html
+"""
+
+import argparse
+import logging
+
+import bct
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, add_verbose_arg,
+                             assert_inputs_exist,
+                             load_matrix_in_any_format)
+
+
+EPILOG = """
+References:
+    [1] Rubinov, Mikail, and Olaf Sporns. "Complex network measures of brain
+        connectivity: uses and interpretations." Neuroimage 52.3 (2010):
+        1059-1069.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        epilog=EPILOG)
+
+    p.add_argument('--in_g1', nargs='+', required=True,
+                   help='')
+    p.add_argument('--in_g2', nargs='+', required=True,
+                   help='')
```

Comment:
Can you define above what is G1 and G2 above please. (Group1 and Group2) I assume.